packages/e2e-tests/cypress/support/e2e.ts

```diff
@@ -0,0 +1 @@
+import "@replayio/cypress/support";
```

Comment:
What's this file for?

Reply:
Assists in capturing the test steps for the cypress test.

Reply:
I don't understand what you mean.

Can you add some inline comments to this file maybe? I would imagine others seeing this and also being confused by it too.

Reply:
I can but I don't expect it to be too foreign for anyone accustomed to how cypress configuration happens. Which, to be fair, I wasn't until all of this fun started but here we are!

Reply:
A module that does nothing but _import_ another module (no exports, no code) _looks_ like a mistake. If I saw this module in isolation, I would assume it was a mistake and raise a PR to delete it.

Given that I'm probably not the only one who would be confused by this in our code base, I think it's probably best to leave some comments as to what it's doing?

Reply:
> I don't expect it to be too foreign for anyone accustomed to how cypress configuration happens

Put another way– who on the frontend team would actually accustomed to how Cypress works enough to understand this? (Not me.)